examples/tracingpolicy/lsm_track_grandparent.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "lsm"
+spec:
+  lsmhooks:
+  - hook: "bprm_check_security"
+    args:
+      - index: 0
+        type: "linux_binprm"
+        extractParam: "mm.owner.real_parent.real_parent.comm"
+        overwriteType: "string"
```

Comment:
do we need overwriteType? could we maybe just do something like:

```
 - index: 0
   type: "string"
   extractParam: "linux_binprm.file.f_path.dentry.d_name.name"
```
the overwrite stuff might cause headaches later

also maybe rename `extractParam` to `resolve`? not sure ;-) 

cc @kkourt

Reply:
I agree, but fyi it seems it's motivated by this https://github.com/cilium/tetragon/pull/3143/files#diff-3c0b10317f585c0016d0208630c3abbb0e8d39571e91254af88874710903a260R442-R455.